### PR TITLE
Update on slides.md file - Issue 302

### PR DESCRIPTION
**Pull request by LavanyaGopal**

### CODE REVIEW

**Review by rachanavishwanath — COMMENTED** [review]
Please squash your commits

**Comment by emilie-robichaud** [comment]
Hi! You should not include the folder **.idea** when you're pushing changes. Can you drop the commit where you have those changes?

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    </ADDITIONAL_INDENT_OPTIONS>
+    <ADDITIONAL_INDENT_OPTIONS fileType="haml">
+      <option name="INDENT_SIZE" value="2" />
+    </ADDITIONAL_INDENT_OPTIONS>
+    <codeStyleSettings language="Groovy">
+      <option name="KEEP_CONTROL_STATEMENT_IN_ONE_LINE" value="false" />
+      <option name="KEEP_BLANK_LINES_IN_DECLARATIONS" value="1" />
+      <option name="KEEP_BLANK_LINES_IN_CODE" value="1" />
+      <option name="KEEP_BLANK_LINES_BEFORE_RBRACE" value="1" />
+      <option name="BRACE_STYLE" value="2" />
```
Comment (rachanavishwanath) [inline .idea/codeStyles/Project.xml:66]: .idea changes are not meant to be commited

File: docs/api-design/slides.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ What is Eclipse Collections?
 * An open source Java Collections Framework
   * Functional, Fluent, Friendly and Fun
   * A mature library with an evolutionary design
-  * 12+ years of development
+  * 18+ years of development
```
Comment (rachanavishwanath) [inline docs/api-design/slides.md:39]: LGTM otherwise! Please remove .idea/* changes and push for another review

**Pull request closed without merging**